### PR TITLE
New Liveness autoconfigure health endpoint

### PR DESCRIPTION
**Pull request by KevinJCross**

### CODE REVIEW

**Comment by KevinJCross** [comment]
> Can you separate out setup from code please, smaller PR etc

yup ... sorry I based it off the previous pr and left the destination as master. 
Same pr  is now pointing to the updated master 😄

File: README.md [hunk 1]
```diff
@@ -3,15 +3,34 @@
 [![Build Status](https://travis-ci.com/hmcts/health-spring-boot-starter.svg?branch=master)](https://travis-ci.com/hmcts/health-spring-boot-starter)
 [ ![Download](https://api.bintray.com/packages/hmcts/hmcts-maven/health-spring-boot-starter/images/download.svg) ](https://bintray.com/hmcts/hmcts-maven/health-spring-boot-starter/_latestVersion)
 
-This library is the autoconfiguration library for the globally health endpoints needed for 
-
-## Getting started
+This is a Spring Boot _starter_ library whose purpose is to autoconfigure any custom HMCTS actuator health endpoints.
+Currently this library contains only the liveness _NOP_ health check needed for Kubernetes to properly operate without restarting the pod.
```
Comment (timja) [inline README.md]:
```suggestion
Currently this library contains only the liveness _NO-OP_ health check needed for Kubernetes to properly operate without restarting the pod. 
```

File: README.md [hunk 2]
```diff
@@ -3,15 +3,34 @@
...
-This library is the autoconfiguration library for the globally health endpoints needed for 
-
-## Getting started
+This is a Spring Boot _starter_ library whose purpose is to autoconfigure any custom HMCTS actuator health endpoints.
+Currently this library contains only the liveness _NOP_ health check needed for Kubernetes to properly operate without restarting the pod. 
+The intention for this library is for any extra organisationally customised health to be implemented here and auto-configured for the projects.
+ 
+Currently this module automatically adds the liveness health detail item which then can be access via `[managment]/health/liveness`
+Where:
+- `[managment]` is where the actuator endpoints are configured. Typically in HMCTS we configure it to `/`
```
Comment (timja) [inline README.md]:
```suggestion
- `[management]` is where the actuator endpoints are configured. Typically in HMCTS we configure it to `/`
```

File: README.md [hunk 3]
```diff
@@ -3,15 +3,34 @@
...
 [ ![Download](https://api.bintray.com/packages/hmcts/hmcts-maven/health-spring-boot-starter/images/download.svg) ](https://bintray.com/hmcts/hmcts-maven/health-spring-boot-starter/_latestVersion)
 
-This library is the autoconfiguration library for the globally health endpoints needed for 
-
-## Getting started
+This is a Spring Boot _starter_ library whose purpose is to autoconfigure any custom HMCTS actuator health endpoints.
+Currently this library contains only the liveness _NOP_ health check needed for Kubernetes to properly operate without restarting the pod. 
+The intention for this library is for any extra organisationally customised health to be implemented here and auto-configured for the projects.
+ 
+Currently this module automatically adds the liveness health detail item which then can be access via `[managment]/health/liveness`
```
Comment (timja) [inline README.md]:
```suggestion
Currently this module automatically adds the liveness health detail item which then can be access via `[management]/health/liveness`
```

Comment (KevinJCross) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (KevinJCross) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (KevinJCross) [reply README.md]: sure

**Comment by KevinJCross** [comment]
> Couple of typo's but other than that good
> 
> Can you also fix this typo please:
> [#1 (comment)](https://github.com/hmcts/health-spring-boot-starter/pull/1#discussion_r255045492)

sure ... done

**Pull request merged**